pkg/sbom/scanner/scanner.go

```diff
@@ -123,7 +123,7 @@ func (s *Scanner) start(ctx context.Context) {
 					scanTimeout = defaultScanTimeout
 				}
 
-				log.Info("scanning from %v, id: %s", reflect.TypeOf(request), request.ID())
+				log.Infof("scanning from %v, id: %s", reflect.TypeOf(request), request.ID())
```

Comment:
I think you can remove this change because of https://github.com/DataDog/datadog-agent/pull/17086